### PR TITLE
add my profile

### PR DESCRIPTION
**Pull request by rb-mwinzinger**
# Create crew member

* added a profile to this repository

### CODE REVIEW

**Comment by peterdey** [comment]
Not sure we need a bookwriter on the crew.

File: Michael/profile.tf [hunk 1]
```diff
@@ -0,0 +1,5 @@
+resource "spaceship_crew_member" "light_yagami" {
+  name        = "Light Yagami"
+  experience  = "15 years"
+  specialty   = "Bookwriter"
```
Comment (peterdey) [inline Michael/profile.tf]: Can we change this to a photographer?

Comment (rb-mwinzinger) [reply Michael/profile.tf]: done. feel frew to review again

**Review by peterdey — APPROVED** [review]
ok for me now

**Review by peuxn — APPROVED** [review]
good

**Pull request merged**